shared/datastore.go

```diff
@@ -14,6 +14,10 @@ import (
 // ErrEntityAlreadyExists is returned by Datastore.Insert when the entity already exists.
 var ErrEntityAlreadyExists = errors.New("datastore: entity already exists")
 
+// MaxKeysPerLookup is the max number of keys allowed per lookup (e.g. GetMulti).
+// https://cloud.google.com/datastore/docs/concepts/limits
```

Comment:
This link sends me to `Firestore in Datastore mode limits` page. I don't think we use Firestore?

Reply:
No we don't use native Firestore. However, https://cloud.google.com/datastore/docs/firestore-or-datastore#automatic_upgrade_to_datastore_mode so this is the technically correct term for Datastore now.